[
    {
        "title": "fix: crash with --parallel and --retries both enabled"
    },
    {
        "author": {
            "login": "PR Description"
        },
        "body": "## PR Checklist\r\n\r\n- [x] Addresses an existing open issue: fixes #5170\r\n- [x] That issue was marked as [`status: accepting prs`](https://github.com/mochajs/mocha/issues?q=is%3Aopen+is%3Aissue+label%3A%22status%3A+accepting+prs%22)\r\n- [x] Steps in [CONTRIBUTING.md](https://github.com/mochajs/mocha/blob/main/.github/CONTRIBUTING.md) were taken\r\n\r\n## Overview\r\n\r\nTurns out there are no existing tests for the case of having both `--parallel` and `--retries` enabled. The problem is that the new circular-reference-breaking logic from #5032 was also being run on other properties in `result` - not just `result.error`."
    },
    {
        "author": {
            "login": "ReDemoNBR"
        },
        "body": "I created https://github.com/mochajs/mocha/issues/5170 . I made a patch file with the changes and applied to my project's CI environment which was failing at all times with 10.6.0 and they all are passing with the patch applied.\r\nLGTM"
    },
    {
        "author": {
            "login": "Uzlopak"
        },
        "body": "LGTM"
    },
    {
        "author": {
            "login": "ludo550"
        },
        "body": "Can this please be merged soon?"
    }
]